apps/src/lib/levelbuilder/lesson-editor/ProgrammingExpressionsEditor.jsx

```diff
@@ -155,6 +162,19 @@ class ProgrammingExpressionsEditor extends Component {
     this.handleRemoveProgrammingExpressionDialogClose();
   };
 
+  handleOpenAddProgrammingExpression = event => {
+    // Prevents button from trigger save
```

Comment:
if we specify that the button element is `type="button"` rather than `type="submit"`, that should prevent this for us; see https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/button-has-type.md

Reply:
Awesome! I did not know this and it will be so helpful going forward

Reply:
When I tested this locally it did not keep the button from triggering the save

Reply:
weird. Fine to leave this for now in the interest of prompt deployment, but we should come back later and figure out what's up here and with some of the other instances of `preventDefault` I've seen in this space

Reply:
https://codedotorg.atlassian.net/browse/PLAT-895